src/runtime/graph/graph_runtime.cc

```diff
@@ -107,7 +123,24 @@ class GraphRuntime : public ModuleNode {
     uint32_t eid = this->entry_id(outputs_[index]);
     TVM_CCALL(TVMArrayCopyFromTo(&data_entry_[eid], data_out, nullptr));
   }
+#ifdef TVM_GRAPH_RUNTIME_DEBUG
+  /*!
+   * \brief Copy index-th node to data_out.
+   * \param index: The  index of the node.
+   * \param data_out the node data.
+   */
+  void DebugGetNodeOutput(int index, DLTensor* data_out) {
```

Comment:
Document in comment that we will do a partial run to the certain node, to avoid it being overriden. This is an costly operation and should only be used for debug

Reply:
Done